types/prompt-sync/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for prompt-sync 4.1.4
-// Project: https://github.com/0x00A/prompt-sync
-// Definitions by: TANAKA Koichi <https://github.com/MugeSo>
+// Type definitions for prompt-sync 4.2.0
+// Project: https://github.com/heapwolf/prompt-sync
+// Definitions by: TANAKA Koichi <https://github.com/MugeSo>, Yingbo Qiu <https://github.com/qyb>
```

Comment:
Are you looking to become a reviewer of PRs to this package, or just contribution credit? If the latter, you shouldn't edit this section: https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners